File: Python/uberBlack.py
from car import Car

class UberX(Car):
    typerCarAccepted = []
    seatsMaterial = []

    def __init__(self, licence, driver, typerCarAccepted, seatsMaterial):
        super.__init__(licence, driver)
        self.typerCarAccepted = typerCarAccepted
        self.seatsMaterial = seatsMaterial

File: Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    Map<String, Map<String, Integer>> typeCarsAccepted;
    ArrayList<String> seatsMaterial;
    
    public UberBlack(String license, Account driver, Map<String, Map<String, Integer>> typeCarsAccepted, 
            ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarsAccepted = typeCarsAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}
